locales/fluent/base/pl.ftl

```diff
@@ -0,0 +1,149 @@
+### Base localization / generic strings
+### Messages should only go into this file if they are widely used,
+### or are particularly important to localize.
+
+## Terms
+
+-service-name = Wikijump
+
+## Special
+
+# Shows whenever a message is still being loaded
+message-loading = Ładowanie...
+
+goto-home = Wróć na stronę główną
+
+goto-service = Wejdź na { -service-name }
+
+base-title = { $title } | { -service-name }
+
+navigated-to = Przejdź do { $path }
+
+## Generic
+
+about = O stronie
+account = Konto
+applications = Aplikacje
+avatar = Awatar
+breadcrumbs = Nawigacja okruszkowa
+change = Zmiany
+clear = Wyczyść
+close = Zamknij
+dashboard = Dashboard
+docs = Dokumentacja
+download = Pobierz
+edit = Edytuj
+editor = Edytor
+footer = Stopka strony
+general = Główne
+header = Nagłówek strony
+help = Pomoc
+inbox = Skrzynka odbiorcza
+invitations = Zaproszenia
+license = Licencja
+load = Ładuj
+main-content = Główna zawartość
+messages = Wiadomość
+navigation = Nawigacja
+notifications = Powiadomienia
+preview = Podgląd
+privacy = Prywatność
+profile = Profil
+publish = Publikacja
+reveal-sidebar = Rozwiń pasek boczny
+save = Zapisz
+security = Bezpieczeństwo
+send = Wyślij
+sent = Wysłano
+settings = Ustawienia
+sidebar = Pasek boczny
+tags = Tagi
+terms = Warunki
+upload = Przekaż plik
+
+search = Szukaj
+  .placeholder = Wyszukiwanie...
+
+## Generic Authentication
+
+login = Zaloguj się
+  .toast = Zostałeś zalogowany.
+
+logout = Wyloguj się
+  .toast = Zostałeś wylogowany.
+
+register = Zarejestruj się
+  .toast = Utworzyłeś konto.
+
+specifier = Email lub Nazwa Użytkownika
+  .placeholder = Wprowadź email lub nazwę użytkownika...
+
+username = Nazwa użytkownika
+  .placeholder = Wprowadź nazwę użytkownika...
+  .info = Możesz to później zmienić.
+
+email = Email
+  .placeholder = Wprowadź adres email...
+  .info = Twój adres email jest prywatny.
+
+password = Hasło
+  .placeholder = Wprowadź hasło...
+
+confirm-password = Potwierdź hasło
+
+forgot-password = Zapomniałem hasła
+  .question = Zapomniałeś hasła?
+
+reset-password = Resetuj hasło
+
+remember-me = Zapamiętaj mnie
+
+create-account = Utwórz konto
+
+field-required = To pole jest wymagane
+
+characters-left = { $count ->
+  [1] Pozostał 1 znak
```

Comment:
Why is this `P` capitalized when the other one is lowecase? Is this a property of Polish I'm unaware of?

Reply:
In english file there's:
```
characters-left = { $count ->
  [1] 1 character left
```
It could be translated as "1 znak pozostał", but it's grammarly incorrect in polish. Sentence "Pozostał 1 znak" sounds better, because verb, not noun, is the first word. Directly translating it to english it would be "Left 1 character" but it's also grammarly incorrect. In polish it's good.

Reply:
Oh, you meant:
```
pozostało { $count } znaków
}
```
Yeah, `P` should be capitalized. It's a mistake, thanks for noticing that